### PR TITLE
feat: resource view grid item description tooltip

### PR DESCRIPTION
**Pull request by jnfrati**
Closes: #4831 

### Description:
ResourceViewGrid[Chart|Dashboard]Items were not showing the description tooltip on hover as the list items were. 
Now when the user hovers at the top group of the card (where the title is) the tooltip with the description is shown.

Lightdash - Google Chrome - 20 March 2023

### CODE REVIEW

File: packages/frontend/src/components/common/ResourceView/ResourceViewGrid/ResourceViewGridChartItem.tsx [hunk 1]
```diff
@@ -40,24 +40,34 @@ const ResourceViewGridChartItem: FC<ResourceViewGridChartItemProps> = ({
...
-                    borderBottomWidth: 1,
-                    borderBottomStyle: 'solid',
-                    borderBottomColor: theme.colors.gray[3],
+            <Tooltip
+                label={item.data.description}
+                withArrow
+                position="top"
+                styles={{
+                    tooltip: { backgroundColor: theme.colors.dark[6] },
+                    arrow: { backgroundColor: theme.colors.dark[6] },
```
Comment (IrakliJani) [inline packages/frontend/src/components/common/ResourceView/ResourceViewGrid/ResourceViewGridChartItem.tsx:49]: do we really need this?

File: packages/frontend/src/components/common/ResourceView/ResourceViewGrid/ResourceViewGridDashboardItem.tsx [hunk 2]
```diff
@@ -40,24 +40,34 @@ const ResourceViewGridDashboardItem: FC<ResourceViewGridDashboardItemProps> = ({
...
-                    borderBottomStyle: 'solid',
-                    borderBottomColor: theme.colors.gray[3],
+            <Tooltip
+                label={item.data.description}
+                withArrow
+                position="top"
+                styles={{
+                    tooltip: { backgroundColor: theme.colors.dark[6] },
+                    arrow: { backgroundColor: theme.colors.dark[6] },
                 }}
```
Comment (IrakliJani) [inline packages/frontend/src/components/common/ResourceView/ResourceViewGrid/ResourceViewGridDashboardItem.tsx:50]: and this as well

File: packages/frontend/src/components/common/ResourceView/ResourceViewGrid/ResourceViewGridChartItem.tsx (hunk 1 above)
Comment (jnfrati) [reply packages/frontend/src/components/common/ResourceView/ResourceViewGrid/ResourceViewGridChartItem.tsx]: Removed!

File: packages/frontend/src/components/common/ResourceView/ResourceViewGrid/ResourceViewGridDashboardItem.tsx (hunk 2 above)
Comment (jnfrati) [reply packages/frontend/src/components/common/ResourceView/ResourceViewGrid/ResourceViewGridDashboardItem.tsx]: Removed!

**Review by IrakliJani — CHANGES_REQUESTED** [review]
@jnfrati there is an issue where description does not exist. you should pass in `disabled` prop to both tooltips in case description is nullish

CleanShot 2023-03-20 at 20 29 26

**Comment by jnfrati** [comment]
Sorry @IrakliJani! Forgot to test that simple case. 
Already pushed the fix 👍

**Review by IrakliJani — APPROVED** [review]
works great! thanks @jnfrati 🙌 🙌 🙌

**Pull request merged**